go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.7.0
 	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets v0.12.0
 	github.com/Masterminds/semver v1.5.0
-	github.com/Microsoft/go-winio v0.6.1
-	github.com/Microsoft/hcsshim v0.12.0
+	github.com/Microsoft/go-winio v0.6.2
+	github.com/Microsoft/hcsshim v0.12.1-0.20240627163216-200feabd854d
```

Comment:
The reason that `hcsshim.SetNnvManagementMacAddresses()` is not working is because the function is not in the latest release. I will have to import the latest commit to use this function. This can work in the code now but I don't think it can be merged into master before it's formal release right? @paulyufan2

Reply:
Yes we do need this as official release instead of using private build...will check with team